src/nrtk_explorer/library/yaml_transforms.py

```diff
@@ -93,7 +95,7 @@ def create_perturber_instance(klass, kwargs):
 # -----------------------------------------------------------------------------
 
 
-class GenericPerturber:
+class GenericPerturber(ImageTransform):
```

Comment:
Is the inheritance needed for the typing? Otherwise happy to just be a std obj.

Reply:
Not technically needed, but since the app expects to be dealing with a `ImageTransform` object, in case we change the abstract class interface down the line at least we see that we need to update the code of this generated class as well